src/SourceGenerators/Uno.UI.SourceGenerators/DependencyObject/DependencyObjectGenerator.cs

```diff
@@ -247,30 +247,30 @@ private void WriteiOSMoveToSuperView(INamedTypeSymbol typeSymbol, IndentedString
 
 				if (overridesWillMoveToSuperview)
 				{
-					builder.AppendFormatInvariant(@"
+					builder.AppendIndented(@"
```

Comment:
This is a multiline block. I don't think we should be having a "multiline" version of `AppendIndented`, but rather search for the first newline inside the block then switch to the multiline version if there is one. It'll make authoring a lot easier, even if at the expense of a lookup when adding the line.

Reply:
@jeromelaban I wanted to keep `AppendIndented` as fast as possible. That's why I introduced a specific API for multiline.

Regarding authoring, actually most multiline appends are *already* broken (with and without this PR). The passed multi-line verbatim string is already indented in most cases, and we add extra indentation to that. So the authoring was and still an issue. Let me know what do you think.

Reply:
Makes sense, let's keep as is for now then. Thanks!